cmd/brew-test-bot.rb

```diff
@@ -447,6 +447,11 @@ def skip(formula_name)
 
     def satisfied_requirements?(formula, spec, dependency = nil)
       requirements = formula.send(spec).requirements
```

Comment:
@ilovezfs Would be cool to get this in before we start building en masse for 10.13.